helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1251,7 +1251,7 @@ instrumentation:
     #     value: java_value
   nodejs:
     repository: ghcr.io/signalfx/splunk-otel-js/splunk-otel-js
-    tag: v2.15.0
+    tag: v3.0.0
```

Comment:
Both versions v2.16.0 and v3.0.0 were released. Asked around and it was recommend to migrate to use v3.0.0. To do this we have to link/add the [breaking change notes](https://github.com/signalfx/splunk-otel-js/releases/tag/v3.0.0) and update our test golden file values.